content/coming_from_processing_java.md

```diff
@@ -468,7 +466,7 @@ from my_other_file import MyClass, my_function # from other_file.py in the same
 from other_file import *  # everything from other_file.py
 ```
 
-When you are using py5 in *imported mode* you can add a special comment to your other files like explained on this [Importing py5 code](https://py5coding.org/content/importing_py5_code.html#importing-imported-mode-code) section.
+When you are using py5 in *imported mode* with code split in more than one file, you might need to add the special comment, `# PY5 IMPORTED MODE CODE`, to the head of your other files, like explained on this [Importing py5 code](https://py5coding.org/content/importing_py5_code.html#importing-imported-mode-code) section.
```

Comment:
Neat I can add comments to the PR line by line

Just change the link in this line to this:

[](importing_imported_mode_code)

Reply:
In theory I can modify your PR but I can't see how to do that.

Reply:
Hi! I made the change but I don't understand how this can link to a different page (importing_py5_code.html) and then the go to the #importing_imported_mode_code anchor.

Reply:
It has to do with the anchor tag added to the markdown in the other file:

https://github.com/py5coding/py5book/blob/3b4195679f5303921829da3fca8f2977a8b4e642/content/importing_py5_code.md?plain=1#L127

It's one of the fancy markdown things Jupyter book adds for us to use for situations like this.

Reply:
I re-built the site and deployed it to prod.

https://py5coding.org/content/coming_from_processing_java.html

Thanks again for your diligence with the PR!